app/screens/settings-screen/account-id.tsx

```diff
@@ -0,0 +1,90 @@
+import { useAccountScreenQuery } from "@app/graphql/generated"
+import { View } from "react-native"
+import { Text, makeStyles } from "@rneui/themed"
+import { GaloySecondaryButton } from "@app/components/atomic/galoy-secondary-button"
+import { useCallback } from "react"
+import Clipboard from "@react-native-clipboard/clipboard"
+import { toastShow } from "@app/utils/toast"
+import { useI18nContext } from "@app/i18n/i18n-react"
+
+export const AccountId: React.FC = () => {
+  const { data } = useAccountScreenQuery()
+  const { LL } = useI18nContext()
+
+  const styles = useStyles()
+
+  const accountId = data?.me?.id || ""
+  const last6digitsOfAccountId = accountId?.slice(-6).toUpperCase()
+
+  const xs = "********—****—****—****—****** "
```

Comment:
```suggestion
  const xs = "********—****—****—****—******"
```

Reply:
The extra space gives the text some padding. Didn't add a style for that.